mesos_collectd.py

```diff
@@ -30,21 +30,54 @@ def get_stats_string(version):
     return stats_cur
 
 
+# FUNCTION: returns a properly formated string of dimensions from a dictionary
+def _d(d):
+        """Formats a dictionary of key/value pairs as a comma-delimited list of
+        key=value tokens."""
+        return ','.join(['='.join(p) for p in d.items()])
+
+
+# FUNCTION: gets the list of framework metrics based on the version of mesos
+def get_framework_string(version):
+    stats_cur = dict(FRAMEWORK_MESOS.items())
+    return stats_cur
+
+
+# FUNCTION: gets the list of task metrics based on the version of mesos
+def get_task_string(version):
+    stats_cur = dict(TASK_MESOS.items())
+    return stats_cur
+
+
+def get_dimension_string(version):
+    dimensions_cur = dict(DIMENSIONS_MESOS.items())
+    return dimensions_cur
+
+
+def lookup_framework_stat(stat, json, conf):
+    val = dig_it_up(json, get_framework_string(conf['version'])[stat].path)
+    return val
+
+
+def lookup_task_stat(stat, json, conf):
+    val = dig_it_up(json, get_task_string(conf['version'])[stat].path)
+    return val
+
+
 # FUNCTION: Collect stats from JSON result
 def lookup_stat(stat, json, conf):
     val = dig_it_up(json, get_stats_string(conf['version'])[stat].path)
 
     # Check to make sure we have a valid result
     # dig_it_up returns False if no match found
-    if not isinstance(val, bool):
-        return val
-    else:
-        return None
+    return val
```

Comment:
since you changed the dig_it_up to return None instead of False the comment above this line needs to be changed